include/LightGBM/config.h

```diff
@@ -329,6 +330,25 @@ struct Config {
   // desc = **Note**: if balanced bagging is enabled, ``bagging_fraction`` will be ignored
   double neg_bagging_fraction = 1.0;
 
+  // default = 1e-4
+  // check = >0.0
+  // desc = used in MVS boosting. If ``mvs_adaptive == true`` then this value is ignored.
+  // desc = used only in ``mvs``
+  double mvs_lambda = 1e-4;
+
+  // default = false
+  // desc = use adaptive variant of mvs boosting
+  // desc = used only in ``mvs``
+  bool mvs_adaptive = false;
+
+  // default = 256000
+  // check = >0
+  // desc = used in MVS boosting training. If dataset size is greater than ``mvs_max_sequential_size``, then threshold
+  // desc = for MVS is chosen for each thread independently.
```

Comment:
```suggestion
  // desc = used in MVS boosting training. If dataset size is greater than ``mvs_max_sequential_size``, then threshold for MVS is chosen for each thread independently.
```

each `// desc =` line becomes 1 bullet point at https://lightgbm.readthedocs.io/en/latest/Parameters.html, so single sentences shouldn't be broken onto multiple lines this way